### DISCUSSION TITLE
text normalization on address format

### DISCUSSION THREAD

**Discussion by Atefeh197** [NVIDIA/NeMo-text-processing#132] [General]
Hello

I have an address in my text that I need to do text normalization "31, 6545, 230 Avenue Southeast".

Nemo Text normalization considers that as a cardinal and converts as: 

"thirty-one, six five four five, two hundred and thirty Avenue Southeast"

Is there any way to convert the address to the address format?

Or is there any way to change the code based on our address format?

**Comment by ekmb** [comment]
@Atefeh197 , what is the expected output for this input?

**Reply by Atefeh197** [reply]
it considers the address as three cardinals, 

but I need "thirty-one, six five four five, two thirty Avenue Southeast"
